tests/gui/pages/main.spec.js

```diff
@@ -241,4 +250,22 @@ describe('Browser: MainPage', function () {
       })
     })
   })
+
+  describe('page template', function () {
+    let $state
+
+    beforeEach(angular.mock.inject(function (_$state_) {
+      $state = _$state_
+    }))
+
+    it('should match the file contents', function () {
```

Comment:
Is this also a temporary thing, or will it remain after webpack has been integrated?

Reply:
What's in the test file should remain as-is with Webpack. We will only remove the custom HTML `request` handler outside `tests/` as Webpack takes care of that.

Reply:
In that case is it also worth testing the other template files? Or is it safe to assume that as long as one got loaded okay, then all the others will have got loaded okay?

Reply:
Yeah that's a good idea, we could certainly test the other pages in the same manner, we just need to create test files for them. Hopefully there's a nice way to test the modals too.